src/Aspire.Hosting.Azure.WebPubSub/AzureWebPubSubExtensions.cs

```diff
@@ -151,6 +151,7 @@ public static IResourceBuilder<AzureWebPubSubResource> AddAzureWebPubSub(this ID
     /// <param name="builder">The builder for the distributed application.</param>
     /// <param name="hubName">The hub name. Hub name is case-insensitive.</param>
     /// <returns></returns>
+    [Obsolete("This method is obsolete because it has been superceded by the AddHub(IResourceBuilder<AzureWebPubSubResource>, string, string?) overload.", true)]
```

Comment:
Why do we need to Obsolete this? Why can't we just have both overloads?

* `AddHub(this IResourceBuilder<AzureWebPubSubResource> builder, [ResourceName] string hubName)`
* `AddHub(this IResourceBuilder<AzureWebPubSubResource> builder, [ResourceName] string name, string hubName)`

And the top one calls the bottom one passing `hubName` for both `name` and `hubName`.